src/mudlet.pro

```diff
@@ -119,39 +119,31 @@ BUILD_COMMIT_TEST = $$lower($$BUILD_COMMIT_TEST)
 }
 
 
-########################## Version and Build setting ###########################
-# Set the current Mudlet Version, unfortunately the Qt documentation suggests
-# that only a #.#.# form without any other alphanumberic suffixes is required:
+# Set Mudlet version (update in CMakeLists.txt as well)
 VERSION = 4.17.2
 
-# if you are distributing modified code, it would be useful if you
-# put something distinguishing into the MUDLET_VERSION_BUILD environment
-# variable (it should use '-' as the first character) to make identification of
-# the used version simpler
-# Note: the qmake BUILD variable is NOT a built-in one
-# BUILD = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD ) {
-# Possible values are:
-# "-dev" for the development build
-# "-ptb" for the public test build
-# "" for the release build
-# A core dev team member setting things up for a release should comment out the
-# following line - as the app-build.txt file must not contain anything (other
-# than whitespace) for a RELEASE build:
-   # BUILD = "-dev-"$${GIT_SHA1}
-   BUILD = ""
+# Set BUILD based on MUDLET_VERSION_BUILD or default
+isEmpty(BUILD) {
+    # Uncomment for development or public test builds:
+    # BUILD = "-dev-"$${GIT_SHA1}
+    # BUILD = "-ptb-"$${GIT_SHA1}
```

Comment:
How are you going to uncomment this for PTBs - you cannot AFAICT as they run this file as it is (**unchanged**) on a scheduled basis (one per night).

Reply:
Looks like we've got a bit of a code labyrinth here :sweat_smile: The previous code wasn't exactly a walk in the park either. Maybe you can help untangle this, or if qmake is causing too much trouble, now might be the time to consider putting it to bed for good.

Reply:
It might just be that the comment is ill-thought - in that it is perhaps saying what might be seen during the CI builds rather than setting it here. OTOH although we still define `APP_BUILD` it is not longer used within the source code so we no longer need to do the equivalent of `#DEFINE APP_BUILD xxxxxx` in the CMake and qmake project files - though those still make use of it in some form.

Reply:
OK, I'll remove this variable to simplify this setup.

Reply:
@SlySven adjusted, have a look.